### PR TITLE
remove some code

### PR DESCRIPTION
**Pull request by ThisIsARealGithubUser**

### CODE REVIEW

**Comment by FlrnDml** [comment]
Why is this PR a thing?

**Review by FlrnDml — COMMENTED** [review]
test

**Review by ThisIsARealGithubUser — COMMENTED** [review]
why

**Review by FlrnDml — APPROVED** [review]
this is fine

**Review by FlrnDml — CHANGES_REQUESTED** [review]
Change it please

File: index.js [hunk 1]
```diff
@@ -4,4 +4,3 @@
 
 console.log("Hi");
 console.log("this is a test");
```
Comment (FlrnDml) [inline index.js:6]: Why

**Review by FlrnDml — APPROVED** [review]

**Review by FlrnDml — APPROVED** [review]
great

**Review by FlrnDml — APPROVED** [review]

**Pull request merged**